packages/synapse-react-client/src/components/TimelinePlot/TimelinePlot.tsx

```diff
@@ -93,6 +92,19 @@ export const TimelinePlot = ({
 
   const { data: eventsData, isLoading } = eventTableQuery
 
+  // filter the phases query response data to the specific species
+  useEffect(() => {
+    if (species) {
+      const phasesForTargetSpecies =
+        hardcodedPhasesQueryResponseData.queryResult?.queryResults.rows.filter(
+          row => {
+            return row.values[phaseSpeciesIndex] == species
+          },
+        )
+      setPhaseData(phasesForTargetSpecies)
+    }
+  }, [species])
```

Comment:
Suggest instead doing `useMemo` and compute the value of `phaseData` instead of making it a state variable, since you never call `setPhaseData` anywhere else.